src/rust/cryptography-x509-verification/src/policy/mod.rs

```diff
@@ -427,6 +436,19 @@ impl<'a, B: CryptoOps> Policy<'a, B> {
 
         self.ca_extension_policy.permits(self, cert, extensions)?;
 
+        // CA/B 6.1.5: Key sizes
+        // NOTE: We don't currently enforce that RSA moduli are divisible by 8,
+        // since other implementations don't bother.
+        let cert_spki = &cert.tbs_cert.spki;
+        if cert_spki.algorithm.is_rsa_key() {
+            let rsa_key: Pksc1RsaPublicKey<'_> =
+                asn1::parse_single(cert_spki.subject_public_key.as_bytes())?;
+
+            if rsa_key.n.as_bytes().len() * 8 < self.minimum_rsa_modulus {
+                return Err(ValidationError::Other("RSA key is too weak".into()));
+            }
+        }
+
```

Comment:
Why not valid issuer with the other key checks?

Reply:
https://github.com/pyca/cryptography/pull/10302#discussion_r1472130273 is my underlying reason, but I think consistency is an equally good reason. Moving!